src/test/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/ccd/EventTest.java

```diff
@@ -38,6 +38,6 @@ public void has_correct_values() {
 
     @Test
     public void if_this_test_fails_it_is_because_it_needs_updating_with_your_changes() {
-        assertEquals(25, Event.values().length);
+        assertEquals(27, Event.values().length);
```

Comment:
You added one Event entry, but increment by two the assert. Maybe is it issue with Github preview, because you rebased on other branches